### ISSUE TITLE
Feature request: cross_zone_load_balancing support for aws_lb

### ISSUE THREAD

**Issue by nodesocket** [terraform-providers/terraform-provider-aws#3574]
Can you add the following new directive and support for `cross_zone_load_balancing` in the [aws_lb](https://www.terraform.io/docs/providers/aws/r/lb.html) resource? This was recently released https://aws.amazon.com/about-aws/whats-new/2018/02/network-load-balancer-now-supports-cross-zone-load-balancing/.

**Comment by bflad** [comment]
You'll be happy to know there is an open PR already. 😄
https://github.com/terraform-providers/terraform-provider-aws/pull/3537

I'll mark it to close this ticket when its merged.

**Comment by bflad** [comment]
Support has been merged into master and will release with v1.11.0 of the AWS provider, likely in about a week. 🎉

**Comment by nodesocket** [comment]
Awesome. Thanks. 🍾

**Comment by bflad** [comment]
This has been released in [version 1.11.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1110-march-09-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

**Comment by israel-tk** [comment]
This is somewhat causing issues for me. I'm using an ALB, which does not support this attribute, but Terraform still says:

```
  ~ module.app.aws_alb.internal_load_balancer
      enable_cross_zone_load_balancing: "" => "false"
```
See #4168.

Shouldn't Terraform ignore this attribute for ALBs?

**Comment by israel-tk** [comment]
I'm using AWS plugin terraform-provider-aws_v1.13.0_x4 and terraform 0.10.8.

This is the relevant part of the config (slightly edited for privacy reasons):
```
resource "aws_alb" "internal_load_balancer" {
  name     = "${var.cluster_name}-intelb-app"
  internal = "true"
  security_groups = ["${var.app_sg_id}"]
  subnets = ["${var.app_subnet1_id}",
             "${var.app_subnet2_id}"]
  enable_deletion_protection = false

  tags {
    Name = "${var.cluster_name}-intelb-app"
  }

  lifecycle {
    prevent_destroy = true
  }
}
```

**Comment by bflad** [comment]
The fix for suppressing non-applicable attributes in aws_lb diffs was made in #4032 and released in [version 1.14.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1140-april-06-2018).

**Comment by israel-tk** [comment]
Awesome, thanks a lot and sorry for the noise!
